assignments/homework/pcosta/497HW1.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import units as u
+from astropy.units import imperial
+from astropy import constants as const
+from astropy.time import Time
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import EarthLocation, AltAz
+from random import *
+from astropy.coordinates import FK5
+
+#Units Notebook Homework
+#1-Unit Conversions
+velocities = np.array([2.3, 3.5, 6.0, 10.5])*u.m/u.s
+convertedVelocities = velocities.to(imperial.mi/u.h)
+print(convertedVelocities)
+
+#2-Using Equivalencies
+specDensity1 = (12 * u.mJy).to(u.erg / u.cm**2 / u.s / u.Hz)
+print(specDensity1)
+
+specDensity2 = (12 * u.mJy).to(u.W / u.m**2 / u.Hz)
+print(specDensity2)
+
+#3-Plotting the Solar System Potential
+def V(r):
+    return -1 * const.G * const.M_sun / r
+R = ((np.arange(1, 50, 0.5))*u.AU)
+plt.plot(R, V(R).to(u.mJ/u.kg))
+plt.title("Gravitational Potential")
+plt.xlabel("Radius (AU)")
+plt.ylabel("Potential (MJ/Kg)")
```

Comment:
if you use the Quantity plotting support then matplotlib becomes aware of units, and will use appropriate axes labels, so you don't set the labels manually.

To enable it, you do the following only once but before you plot the first plot.
```
from astropy.visualization import quantity_support
quantity_support()
```

https://docs.astropy.org/en/stable/visualization/matplotlib_integration.html#plotting-quantities